### DISCUSSION TITLE
Can't build android app

### DISCUSSION THREAD

**Discussion by TharinduKetipe** [software-mansion/react-native-reanimated#3231] [Q&A]
Hi! I'm getting following error since last Friday

```
Configure project :react-native-reanimated
No AAR for react-native-reanimated found. Attempting to build from source.
Native libs debug enabled: false
Android gradle plugin: 4.0.1
Gradle: 6.3
building Reanimated2
```
I have tried millions of solutions whole day. But none of them weren't help to fix this issue.
Library version : 2.8.0
Gradle version : 6.3-all
react : 16.13.1
react-native: 0.68.2

**Comment by mr-574rk** [comment]
Im still getting this error till date, did you manage to fix it?

**Comment by nghiadqdev** [comment]
:(( I just get this error and look like not have an issue to fix
